board/management/commands/getgamefromapi.py

```diff
@@ -19,9 +19,9 @@ def handle(self, *args, **options):
         self._base_init()
         game_id = options['id']
 
-        response = igdb_wrapper.get_game_by_id(game_id)
-        if response:
-            game = GameAPI(response['id'])
+        game = GameAPI(game_id)
+        if game.is_empty():
+            game = GameAPI(game_id)
```

Comment:
I guess this doesn't solve the problem with two requests, just moves one call to `GameAPI` `__init__` method.